demo/src/app/components/alert/demos/basic/basic.alert.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+
+import { NgbAlert } from '@ng-bootstrap/alert';
```

Comment:
@pkozlowski-opensource  It will work. adding this in webpack.demo is enough I guess.